### PR TITLE
Article to dev

### PR DESCRIPTION
**Pull request by LenaIks**
with search option

### CODE REVIEW

File: src/components/US-01-HotTracks/components/ArticleCard/index.js [hunk 1]
```diff
@@ -0,0 +1,50 @@
+import React, {useState, useEffect}from 'react'
+import './style.css'
+import ArticleContent from'../ArticleContent'
+import Axios from 'Axios'
+
+
+export default function ArticleCard(props) {
+    const  {index,title, creator,description, content} = props;
```
Comment (EducationWildCodeSchool) [inline src/components/US-01-HotTracks/components/ArticleCard/index.js:8]: you can set this in line 7 like: 

export default function ArticleCard( {index, title, creator,description,content} )

File: src/components/US-01-HotTracks/components/ArticleCard/index.js [hunk 2]
```diff
@@ -0,0 +1,50 @@
...
+    const [contentExist, setContentExist] = useState(false);
+    const [showContent, setShowContent]= useState(false);
+    const [isFavorite, handleClickFavorite] = useState(false); 
+   
+
+    useEffect(()=>{ if(content != null){setContentExist(true)}
```
Comment (EducationWildCodeSchool) [inline src/components/US-01-HotTracks/components/ArticleCard/index.js:14]: you don't need this state, the existence of content is determined by the props.content. So you can ask below `if(content){ // do something }`

File: src/components/US-01-HotTracks/index.js [hunk 3]
```diff
@@ -1,42 +1,65 @@
...
 import './style.css'
 
 
 export default function HotTracks() {
+  const [searchValue, setSearchValue] = useState("");
+  const [searchButton, setSearchButton]= useState (false);
 
-   
-    const [data, setData] = useState([]);
+   const key= "pub_209372810a042a82b26699a3eba51ed7ef6c";
```
Comment (EducationWildCodeSchool) [inline src/components/US-01-HotTracks/index.js:12]: you can put this key in an env variable

File: src/components/US-01-HotTracks/index.js [hunk 4]
```diff
@@ -1,42 +1,65 @@
...
         
-        </div>
-
-      );
+       
+    
+    
+          return (
+            <div className="articleOverview">
+                {console.log(data)}
```
Comment (EducationWildCodeSchool) [inline src/components/US-01-HotTracks/index.js:44]: you can remove this :)

**Review by EducationWildCodeSchool — COMMENTED** [review]
Good job! Leaving some suggestions

**Comment by LenaIks** [comment]
merged to dev

**Pull request merged**